ModelInstaller.iss

```diff
@@ -34,144 +34,15 @@ Name: "english"; MessagesFile: "compiler:Default.isl"
 Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"; Flags: unchecked
 
 [Files]
-Source: "C:\Users\Zunyue\Desktop\cs561_project\ModelViewer\ModelViewer\x64\Release\ModelViewerRun.exe"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\advapi32.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\advapi32res.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\combase.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\cryptbase.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\d3d11.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\dwmapi.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\dxgi.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\gdi32.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\gdi32full.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\imm32.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\kernel32.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\KernelBase.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\mpr.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\mprapi.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\mprddm.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\mprdim.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\mprext.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\msv1_0.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\msvbvm60.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\msvcirt.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "C:\Windows\System32\msvcp_win.dll"; DestDir: "{app}"; Flags: ignoreversion

```

Comment:
Since we're packaging a compiled version of our code, we don't have to include any compilers :)